backend/common/src/main/java/ai/verta/modeldb/common/CommonConstants.java

```diff
@@ -13,6 +13,10 @@ public interface CommonConstants {
   String EMPTY_STRING = "";
   String S3 = "S3";
   Integer TAG_LENGTH = 40;
+  Integer THREAD_COUNT = 256;
```

Comment:
recommend naming this with its purpose. Is this the thread count for ... grpc clients? grpc servers? internal futures?

Reply:
currently not in used so removed it